ceph.spec.in

```diff
@@ -846,7 +844,7 @@ mkdir -p %{_localstatedir}/run/ceph/
 %{_datadir}/ceph/id_dsa_drop.ceph.com
 %{_datadir}/ceph/id_dsa_drop.ceph.com.pub
 %dir %{_sysconfdir}/ceph/
-%dir %{_localstatedir}/log/ceph/
+%attr(3770,ceph,ceph) %dir %{_localstatedir}/log/ceph/
```

Comment:
This would cause conflicts as the same dir is specified in the file list twice and with different perms (see the last chunk of this patch).